cmd/cli/upsert.go

```diff
@@ -58,4 +58,6 @@ func init() {
 	upsertCmd.Flags().DurationVar(&upsertArgs.MinRetryTime, "retry-min-time", time.Millisecond*200, "Minimum wait interval")
 	upsertCmd.Flags().DurationVar(&upsertArgs.MaxRetryTime, "retry-max-time", time.Second*30, "Maximum wait interval")
 	upsertCmd.Flags().IntVar(&upsertArgs.MaxRetryCount, "retry-max-count", 12, "Maximum number of retries before giving up")
+	upsertCmd.Flags().BoolVar(&upsertArgs.AppendGroups, "append-groups", false, "append to a existing group list")
```

Comment:
Would it be more concise if we just use `--append` flag - given that groups is the only list item in the map, that way it would look like:
```
aws-auth upsert  --maproles --rolearn arn:aws:iam::00000000000:role/test  --username test  --groups test --append
```

WDYT?

Reply:
No problem, I figured `--append-groups` was clearer on what its for. I'll update it.